docs/streams/developer-guide/processor-api.html

```diff
@@ -428,12 +463,20 @@ <h2><a class="toc-backref" href="#id8">Connecting Processors and State Stores</a
 builder.addSource("Source", "source-topic")
     // add the WordCountProcessor node which takes the source processor as its upstream processor.
     // the ProcessorSupplier provides the count store associated with the WordCountProcessor
-    .addProcessor("Process", new ProcessorSupplier&ltString, String&gt() {
-        public Processor&ltString, String&gt get() {
+    .addProcessor("Process", new ProcessorSupplier&lt;String, String, String, String&gt;() {
+        public Processor&lt;String, String, String, String&gt; get() {
             return new WordCountProcessor();
         }
-        public Set&ltStoreBuilder&lt?&gt&gt stores() {
-            return countStoreBuilder;
+
+        public Set&lt;StoreBuilder&lt;?&gt;&gt; stores() {
+            final StoreBuilder&lt;KeyValueStore&lt;String, Long&gt;&gt; countsStoreBuilder =
+                Stores
+                    .keyValueStoreBuilder(
+                        Stores.persistentKeyValueStore("Counts"),
+                        Serdes.String(),
+                        Serdes.Long()
+                    );
+            return Collections.singleton(countsStoreBuilder);
```

Comment:
For this example, it seems more appropriate to depict a self-contained store definition, rather than referencing an externally defined store builder, as in the other example.